usefulUtilities/tabStatus/appResources/appData/resources/js/tabStatus_ui.js

```diff
@@ -0,0 +1,606 @@
+        teamname: teamname
+    })
+}
+
+
+// Handle if someone clicked on the headers to sort
+function handleTableClick(event){
+    let type;
+    if (event.target.tagName === "TH") {
+        type = event.target.innerHTML;
+        if (previousSortType) {
+            previousSortType = sortType;
+        }
+        sortType = type;
+        sortArray();
+        filterMembers();
+        previousSortType = sortType; 
+        sendSortState();
+    }
+}
+
+
+// Get out of tutorial mode
+function handleGotIt(){
+    emitAppSpecificEvent("onGotItClicked")
+}
+
+// #endregion
+// *************************************
+// END handlers
+// *************************************
+
+// *************************************
+// START date-transfers
+// *************************************
+// #region date-transfers
+
+
+// Emit an event specific to the `multiConApp` over the EventBridge.
+var APP_NAME = "tabStatus";
+function emitAppSpecificEvent(method, data) {
+    var event = {
+        app: APP_NAME,
+        method: method,
+        data: data
+    };
+    EventBridge.emitWebEvent(JSON.stringify(event));
+}
+
+
+// send sort related details to save state
+let currentlySearching = false;
+let currentlySorted = false;
+let sortType = null;
+let previousSortType = null;
+function sendSortState(){
+    emitAppSpecificEvent("onSortSettingsChange", {
+        sortType: sortType,
+        currentlySorted: currentlySorted,
+        currentlySearching: currentlySearching,
+        previousSortType: previousSortType
+    })    
+}
+
+
+// Get the actual employee data
+let allPeople = [];
+const REFRESH_TIME_MS = 10000;
+function getEmployeeData(){
+    fetch('https://highfidelity.co/api/statusIndicator/?type=getAllEmployees')
+    .then(response => { 
+        return response.json() 
+    })
+    .then(data => {
+        allPeople = [];
+        data.teams.forEach(team => {
+            team.members.forEach(member =>{
+                allPeople.push(
+                    new WorkerMaker(
+                        member.displayName, 
+                        member.status, 
+                        member.location,
+                        team.name
+                    )
+                )
+            })
+        })
+        setTimeout(getEmployeeData, REFRESH_TIME_MS);
+        renderUI();
+    })
+    .catch(error => { console.error(error) })
+}
+
+
+// #endregion
+// *************************************
+// END date-transfers
+// *************************************
+
+// *************************************
+// START render
+// *************************************
+// #region render
+
+
+// handles rendering differnt ui states
+function renderUI(){
+    if (currentlySorted){
+        // handles making sure the sort order is the correct direction
+        if (previousSortType === sortType){
+            previousSortType = null;
+            sendSortState();
+            sortArray();
+        } else {
+            sortArray();

```

Comment:
looks like you can pull this call out of the conditional since you're doing it no matter what, right?